perimeter/docs/resource_policy_shared_accounts_access.md

```diff
@@ -0,0 +1,68 @@
+This benchmark answers the following questions:
+
+- What resources have resource policies that allow untrusted cross account access?
```

Comment:
```suggestion
- What resources have resource policies that allow untrusted cross-account access?
```

I believe AWS usually hyphenates `cross-account`, e.g., https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/, so we should follow the same.

Can you please update all references in docs and `.sp` file descriptions + titles?